.github/workflows/jest-test-and-lint.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     - name: Checkout Branch
       uses: actions/checkout@v2
     - name: Prepare npm
-      uses: parachutehome/workflows/.github/actions/npm-prepare@trunk
+      uses: parachutehome/workflows/.github/actions/npm-prepare@volta
```

Comment:
do we want to target `volta` branch here or `main`?

Reply:
It needs to be volta currently in the PR to test but, yup, should switch to main.

Ideally it should be locked to a version I'm just looking to see if there is an easy way to reference the "current branch".

Reply:
Did a really quick / dirty script to run when we create a version branch.